lib/stack_master/commands/init.rb

```diff
@@ -24,8 +24,8 @@ def perform
       def check_files
         @stack_master_filename = "stack_master.yml"
         @stack_json_filename = "templates/#{@stack_name}.json"
-        @parameters_filename = File.join("parameters", "#{underscored_stack_name}.yml")
-        @region_parameters_filename = File.join("parameters", @region, "#{underscored_stack_name}.yml")
+        @parameters_filename = File.join("parameters", "#{@stack_name}.yml")
+        @region_parameters_filename = File.join("parameters", @region, "#{@stack_name}.yml")
```

Comment:
`stack_master init` uses the stack name as provided on the CLI when naming parameter files.